cmake/apple/install_apple.sh.in

```diff
@@ -21,20 +21,16 @@ MSG_COLOR='\x1B[1;36m'
 COLOR_RESET='\x1B[0m'
 echo -e "$MSG_COLOR\n\nCreating App Bundle \"$APP\"...$COLOR_RESET"
 
-# Locate macdeployqt, assume homebrew & Qt5
-which macdeployqt > /dev/null 2>&1
```

Comment:
@tresf, I'm done with school exams so now I've got time for this. :)

So, is MacPorts still used ? I've never heard of it so I don't know if it's because it's old or just because I don't use MacOS (while I've heard of homebrew...).

I could drop support for it if it's not used anymore or just fix the script in another way, I finally understood the (hacky) script, so I guess I could keep support for MacPorts.